### DISCUSSION TITLE
Temporarily Inactive

### DISCUSSION THREAD

**Discussion by Hiro-Tokyo** [code4recovery/12-step-meeting-list#703] [Q&A] [answered]
Hello Team,

I’m Hiro from Tokyo. I’m building Japan Online Meeting Website.

In Japan, some of online meetings are using online meeting platform which exist just in Japan, so can not validate with meeting guide TSML Wordpress plug-in. These meetings show up as ‘Temporarily Inactive’ on our website. Any way to replace with green ‘Online’?

I think probably replacing with ‘Online’ in CSS feed could be temporarily solution but it’s not fundamental solution. 

If you can support about it, it would be helpful.

https://aajapan-online-intergroup.org/

This website is not open yet so meeting list is not public yet. It will open on March 1st.

**Comment by joshreisner** [comment, marked as answer]
Hi @Hiro-Tokyo !

Could you tell us more about the online meeting platform? Perhaps we need to add it to the list of authorized meeting providers.

* What's it called?
* Are you putting links to the online meeting platform in your meeting listings?
* What does an example URL look like? eg `https://japan.online.meeting.jp/join/12356789?pwd=12378`

**Reply by Hiro-Tokyo** [reply]
Thanks Josh!

It's called jit.si

I guess it open source meeting platform. I think this is not only in Japan.
Meeting ID will be like this.

https://meet.jit.si/### meeting_name
*They use 2 byte Japanese character for their meeting name, ID.

When I put jit.si link on the meeting list on our website, meeting guide plug-in prevents that link.

And I have one more issue.
Some of groups would like to link to their meeting information website, I mean they would prefer not to provide their zoom meeting links on our website. When the meeting link is empty, Temporarily Inactive show up on these meetings. This is another our issue, We are asking them to provide their meetings links though.

**Reply by joshreisner** [reply]
Ah great, we can see about adding Jitsi to the authorized list of providers. We'll want to coordinate that with Meeting Guide. But in the meantime, do you have access to your theme's functions.php file? Try adding this code to it:

```
if (!empty($tsml_conference_providers)) {
	$tsml_conference_providers += [
		'meet.jit.si' => 'Jitsi',
	];
}
```

This will add them as an authorized provider to your site. 

As for the groups who want to link to their own websites, it would be good to push back on them to improve the user experience for the majority of users. You technically could add their domain as an authorized provider as above, but if you do that they'll have no incentive to change 🙂

Also for security reasons Meeting Guide will have to treat them as inactive, if you connect.

```
if (!empty($tsml_conference_providers)) {
	$tsml_conference_providers += [
		'meet.jit.si' => 'Jitsi',
		'myrandom.website.jp' => 'One Day at a Time Group,
	];
}
```

**Reply by joshreisner** [reply]
But my suggestion is to either omit them from the listings or leave the "Inactive" flag on them and put their URL under "Group Contact Information" at the bottom.

**Reply by Hiro-Tokyo** [reply]
Thank you so much Josh! That code is great! I will talk these group to provide meeting ID following your suggestion.
